themes/app/templates/Layout/Page.ss

```diff
@@ -1,4 +1,13 @@
-<h2>$Title</h2>
-$Content
-$Form
-$ElementalArea
+$Breadcrumbs
+<div class="container">
+    <div class="row">
+        <div class="typography col-lg-8">
+            <p class="lead page-content__summary">$Summary</p>
+            $Content
+        </div>
+        <div class="offset-xl-1 col-xl-3 col-lg-4">
```

Comment:
I think this can be refactored to `col-md-3` `col-md-offset-1`

Reply:
This is also set on purpose, was told by James to do this.

Reply:
ahh I see, alrighty 👍